assets/css/styles.css

```diff
@@ -0,0 +1,299 @@
+.section-title {
+  color: var(--beige);
+  font-size: 18px;
+  font-weight: 400;
+  text-align: right;
+  padding: 4px 12px;
+  width: 18%;
+  border-right: 8px solid var(--deep-space-sparkle);
+}
+
+/* responsive section containers */
+.aboutme-container,
+.portfolio-container,
+.contact-container {
+  display: flex;
+  flex-direction: row;
+  margin: 18px;
+}
+
+.aboutme-flex-container {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+}
+
+.portfolio-flex-container {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+}
+
+/* about me section*/
+#imgCS {
+  width: 15%;
+  height: auto;
+  object-fit: contain;
+  margin: 18px;
+  border: solid 4px var(--deep-space-sparkle);
+  box-shadow: 18px 18px 18px 18px #a0a0a0 inset;
+}
+
+.aboutme-content {
+  color: var(--deep-space-sparkle);
+  padding: 4px 8px;
+  width: 100%;
+  font-size: 1.64vw;
+}
+
+.aboutme li {
+  list-style-type: none;
+}
+
+.aboutme li strong {
+  color: var(--orange-red-crayola);
+}
+
+#cv {
+  text-decoration: none;
+  color: var(--beige);
+}
+
+/* portfolio section*/
+/* main image */
+.portfolio-main-container {
+  margin: auto;
+  width: 80%;
+  height: auto;
+  padding: 18px 0px;
+}
+
+.portfolio-img1 {
+  width: 100%;
+  height: 150px;
+  padding: 0px;
+  object-fit: cover;
+  object-position: center top;
+  border: 8px solid var(--deep-space-sparkle);
+}
+
+/* titles */
+.portfolio-name {
+  background-color: var(--deep-space-sparkle);
+  color: var(--beige);
+  padding: 1px;
+  margin-top: -8px;
+  font-size: 12px;
+  text-align: center;
+}
+
+.portfolio-name h3 {
+  margin: 4px;
+}
+
+.portfolio-name p {
+  color: var(--orange-red-crayola);
+  margin: 4px;
+}
+
+.portfolio-name i {
+  font-size: 18px;
+  color: var(--cadet-blue);
+}
+
+/* secondary images */
+.portfolio-sub-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.project-card {
+  margin: 18px;
+}
+
+.portfolio-img2 {
+  width: 100%;
+  height: 150px;
+  padding: 0px;
+  margin-top: 8px;
+  margin-bottom: 0px;
+  object-fit: cover;
+  object-position: right top;
+  border: 8px solid var(--deep-space-sparkle);
+}
+
+.portfolio-img2:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/* contact section */
+.contacts {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 18px;
+  width: 80%;
+}
+
+.contacts a {
+  font-size: 32px;
+  margin: 0px auto;
+  color: var(--deep-space-sparkle);
+}
+
+/* footer */
+.footer {
+  background-color: var(--deep-space-sparkle);
+  color: var(--cadet-blue);
+  text-align: center;
+  padding: 8px;
+  width: 100%;
+}
+
+/* mobile viewport styles */
+/* navbar */
+@media screen and (max-width: 576px) {
+  .navbar-container,
+  .nav-link-container {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  /* hero relative to sticky navbar */
+  /* hero image */
+  /* #hero {
+    margin-top: 48%;
+  } */
+
+  /* section containers */
+  .aboutme-container,
```

Comment:
This links to the comment above in the main section - if you rename to consolidate, make sure to update this too